### PR TITLE
Add comment to ex: update k8s version to desired

### PR DESCRIPTION
**Pull request by nfd9001**
Another comment from the same user mentioned in #3486.

### CODE REVIEW

**Comment by PRMerger12** [comment]
@nfd9001 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
@mihaelablendea - please review.

**Comment by Jak-MS** [comment]
@MikeRayMSFT - are you still waiting on a review from @mihaelablendea or is this ready for merge? (if so, can you sign off?) thanks!

File: docs/big-data-cluster/deploy-with-kubeadm.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ On each machine, there are several required prerequisites. In a bash terminal, r
 1. Configure docker and Kubernetes prerequisites on the machine.
 
    ```bash
-   KUBE_DPKG_VERSION=1.15.0-00
+   KUBE_DPKG_VERSION=1.15.0-00 #or your other target K8s version
```
Comment (mihaelablendea) [inline docs/big-data-cluster/deploy-with-kubeadm.md]: Should specify though that minimum is 1.13

**Comment by ktoliver** [comment]
@nfd9001 Could you update the pull request per reviewer comments? Thanks.

**Comment by PRMerger9** [comment]
@nfd9001 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
#sign-off

**Pull request merged**